components/FeedList/FeedList.jsx

```diff
@@ -4,16 +4,26 @@ import {FlatList, View} from 'react-native';
 import Photos from '../../screens/ProfileScreen/page/Photos/Photos';
 import SkeletonPhotos from '../Skeleton/SkeletonPhotos';
 import SkeletonPosts from '../Skeleton/SkeletonPosts';
-import Warning from '../Warning/Warning';
 import FeedItem from './FeedItem';
 import styles from './style';
+import {memo, useMemo, useState} from 'react';
```

Comment:
external imports should be above internal imports

Reply:
Already fixed in all places of the code where it was imported in the wrong place